.travis.yml

```diff
@@ -22,44 +22,30 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env:
-        - DEPS=lowest
-    - php: 5.6
-      env:
-        - DEPS=latest
-    - php: 7
-      env:
-        - DEPS=lowest
-    - php: 7
-      env:
-        - DEPS=latest
-    - php: 7.1
+    - php: 7.3
+      dist: bionic
       env:
         - DEPS=lowest
-    - php: 7.1
+    - php: 7.3
+      dist: bionic
       env:
         - DEPS=latest
         - CS_CHECK=true
         - TEST_COVERAGE=true
-    - php: 7.2
-      env:
-        - DEPS=lowest
-    - php: 7.2
-      env:
-        - DEPS=latest
-    - php: 7.3
+    - php: 7.4
       env:
         - DEPS=lowest
-    - php: 7.3
+    - php: 7.4
       env:
         - DEPS=latest
-    - php: 7.4
+    - php: 8.0
       env:
         - DEPS=lowest
-    - php: 7.4
+        - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
```

Comment:
Does installation work without the `--ignore-platform-reqs` now? If so, would drop it before merging

Reply:
I removed this, and CI still passes - good. So I don't know why the issue had an action point "Modify .travis.yml to ignore platform requirements when installing composer dependencies" - I guess that point is out-of-date.